docs/development.md

```diff
@@ -91,3 +91,103 @@ The `cbindgen` xtask generates multiple header files for four different modules:
 4. The types used by the C++ interpreter API, written to `slint_interpreter_internal.h`.
 
 Typically the input to `cbindgen` is within `ffi` sub-modules in the corresponding input crates to `cbindgen`. These `ffi` modules are gated with `#[cfg(feature = "ffi")]`.
+
+## Commit History & Code Reviews
```

Comment:
another thing worth adding would be how to write good commit message (what to expect in a commit message)
But can be done in another PR